src/NServiceBus.RabbitMQ/AssemblyInfo.cs

```diff
@@ -1,9 +1,3 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
+﻿using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("NServiceBus.RabbitMQ")]
-[assembly: AssemblyCopyright("Copyright NServiceBus. All rights reserved")]
-[assembly: AssemblyProduct("NServiceBus.RabbitMQ")]
```

Comment:
Ah, it looks like `AsssemblyTitle` does work. I guess since they didn't originally intend to expose `Title` as a NuGet property, they created a separate property to handle that directly, sigh. I'll update and add both of those then.